src/test/debuginfo/function-prologue-stepping-no-stack-check.rs

```diff
@@ -37,100 +37,79 @@
 // gdb-command:run
 
 // IMMEDIATE ARGS
-// gdb-command:print a
-// gdb-check:$1 = 1
-// gdb-command:print b
-// gdb-check:$2 = true
-// gdb-command:print c
-// gdb-check:$3 = 2.5
+// gdb-command:info args
```

Comment:
If the `morestack` stuff is gone for good, then this test should deleted, but `function-prologue-stepping-regular.rs` should then also handle GDB. I'm surprised that `info args` and `print` give different results here...